### PR TITLE
HAWQ-269. Change net_disk_ratio from int to double type.

### PR DESCRIPTION
**Pull request by zhangh43**
net_disk_ratio is the punishment factor of the size of remote read compared with local read.
we need to set it to be double type.

### CODE REVIEW

**Comment by yaoj2** [comment]
+1

**Comment by wengyanqing** [comment]
looks good

File: src/backend/utils/misc/guc.c [hunk 1]
```diff
@@ -720,7 +720,7 @@ int			optimizer_log_failure;
 double		optimizer_cost_threshold;
 double		optimizer_nestloop_factor;
 double locality_upper_bound;
-int net_disk_ratio;
+double net_disk_ratio;
```
Comment (huor) [inline src/backend/utils/misc/guc.c]: The indentation needs be followed here

File: src/backend/utils/misc/guc.c [hunk 2]
```diff
@@ -6857,6 +6846,15 @@ static struct config_real ConfigureNamesReal[] =
 		&locality_upper_bound,
 		1.2, 1.0, 2.0, NULL, NULL
 	},
+	{
+		{"net_disk_ratio", PGC_USERSET, DEVELOPER_OPTIONS,
+			gettext_noop("set the scan volumn ratio of disk against network."),
```
Comment (huor) [inline src/backend/utils/misc/guc.c]: "Sets the scan ..."

**Comment by huor** [comment]
The overall fix looks good. +1

File: src/backend/utils/misc/guc.c (hunk 1 above)
Comment (zhangh43) [inline src/backend/utils/misc/guc.c]: I'll change it, Thanks

**Pull request closed without merging**